autogpt_platform/backend/backend/integrations/oauth/__init__.py

```diff
@@ -3,6 +3,7 @@
 from .google import GoogleOAuthHandler
 from .notion import NotionOAuthHandler
 
+# --8<-- [start:HANDLERS_BY_NAMEExample]
```

Comment:
nit: inconsistent; `GoogleOAuthIsConfigured` is lowercase but const is uppercase, here both comment and const are uppercase and `Example` is added. Sometimes there's space after `start: ` and sometimes not; I assume and hope that won't break docs.

Reply:
fixed, also they're kinda named after the location in code(this case), or the example (GoogleOAuthIsConfigured)